### PR TITLE
doc: add minutes for meeting 2017-03-07

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: doc/meetings/build-wg-meeting-2017-03-07.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+* GitHub issue: https://github.com/nodejs/build/issues/640
+* Meeting video: https://www.youtube.com/watch?v=8RhgJMn5R88
+* Previous meeting: https://docs.google.com/document/d/1A4Uv3gYoNxZONbRe6P_h36tphQotkObq8neHIXxJhrg/edit
+* Next meeting: 20 March 2017
+
+## Present
+* @mhdawson   Michael Dawson
+* @joaocgreis João Reis
+* @jgergstroem Johan Bergström
+* @gibfan Gibson Fahnestock
```
Comment (gibfahn) [inline doc/meetings/build-wg-meeting-2017-03-07.md]: `@gibfan` -> `@gibfahn`

File: doc/meetings/build-wg-meeting-2017-03-07.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+* GitHub issue: https://github.com/nodejs/build/issues/640
+* Meeting video: https://www.youtube.com/watch?v=8RhgJMn5R88
+* Previous meeting: https://docs.google.com/document/d/1A4Uv3gYoNxZONbRe6P_h36tphQotkObq8neHIXxJhrg/edit
+* Next meeting: 20 March 2017
+
+## Present
+* @mhdawson   Michael Dawson
+* @joaocgreis João Reis
+* @jgergstroem Johan Bergström
```
Comment (gibfahn) [inline doc/meetings/build-wg-meeting-2017-03-07.md]: `@jgergstroem` -> `@jbergstroem`

File: doc/meetings/build-wg-meeting-2017-03-07.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+* Either remove from agenda or set a date (for change) we work against! (Doesn.t seem to be a high prio for anyone)
+
+### TAP Plugin issues on Jenkins [#453](https://github.com/nodejs/build/issues/453)
+
+* No progress afaik (johan)
+* Johan will try to test out a bit more fully by running on a broader set of machines. 
+* Gibson will also try to use it in some of the other projects like node-report.
+
+## other issues/topics
+  * Did supported platforms PR land ? - No - nodejs/node#8922
```
Comment (gibfahn) [inline doc/meetings/build-wg-meeting-2017-03-07.md]: `nodejs/node#8922` -> `[nodejs/node#8922](https://github.com/nodejs/node/pull/8922)`

File: doc/meetings/build-wg-meeting-2017-03-07.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    *  not yet, lets try to make sure it lands this week or so.
+  * look into rewriting jenkins host init script in systemd for
+    auto-restarting after oom.s
```
Comment (gibfahn) [inline doc/meetings/build-wg-meeting-2017-03-07.md]: `oom.s` -> `OOMs`

File: doc/meetings/build-wg-meeting-2017-03-07.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    * create issue, and mark as .good first contribution (johan)
```
Comment (gibfahn) [inline doc/meetings/build-wg-meeting-2017-03-07.md]: `.good` -> `good`

File: doc/meetings/build-wg-meeting-2017-03-07.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+## Minutes
+
+### file and directory names for downloads [#515](https://github.com/nodejs/build/issues/515)
+
+* No discussions have been made -- remove from agenda?
+
+### Draft text for HSTS communication [#484](https://github.com/nodejs/build/issues/484)
+
+* Either remove from agenda or set a date (for change) we work against! (Doesn.t seem to be a high prio for anyone)
```
Comment (gibfahn) [inline doc/meetings/build-wg-meeting-2017-03-07.md]: `Doesn.t` -> `Doesn't`

**Comment by joaocgreis** [comment]
A suggestion, to be discussed: For minutes (only for minutes!), we could make these corrections directly using GitHub's edit feature, adding commits to the PR. When landing, @mhdawson would simply have to interactive rebase on top of master, reword the first commit to add metadata and apply all others as fixups.

@mhdawson since you are writing the minutes, I guess it's up to you to decide how you prefer. When you open the next minutes PR you could explicitly invite this in the opening comment, if you want.

**Comment by mhdawson** [comment]
@joaocgreis your suggestion sounds good to me.

**Comment by mhdawson** [comment]
Landed as d8afb4d95d609c9fba5359548bb1729146ccfab8

**Pull request closed without merging**